agent/taskresource/cgroup.go

```diff
@@ -0,0 +1,225 @@
+// +build linux
```

Comment:
Would it be better to move this into a subdirectory, as there will be lots of files in the `taskresource` directory if more resource was added, like:
```
agent/taskresource/cgroup/cgroup.go
agent/taskresource/volume/volume.go
agent/taskresource/interface.go
```

Reply:
I shall go ahead and add cgroup into a folder and leave volume as is, since it requires more refactoring, which can be addressed in a separate PR.